#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_link-gcrypt.dpatch by Yavor Doganov <yavor@gnu.org>
##
## DP: Detect libgcrypt and link with it.


@DPATCH@
diff -urNad kazehakase-0.5.4~/configure.ac kazehakase-0.5.4/configure.ac
--- kazehakase-0.5.4~/configure.ac	2008-03-29 10:20:42.000000000 +0200
+++ kazehakase-0.5.4/configure.ac	2008-05-16 00:51:04.000000000 +0300
@@ -171,6 +171,9 @@
    AM_PATH_LIBGNUTLS(1.2.0,
 	AC_DEFINE(USE_SSL, 1, [Define using GNU TLS]),
 	AC_MSG_WARN([[libgnutls was not found.]]))
+
+   AM_PATH_LIBGCRYPT([1.2.0], [],
+   		     [AC_MSG_WARN([libgcrypt was not found.])])
 fi
 AM_CONDITIONAL(ENABLE_SSL, test x"$disable_ssl" = "xno")
 
diff -urNad kazehakase-0.5.4~/src/Makefile.am kazehakase-0.5.4/src/Makefile.am
--- kazehakase-0.5.4~/src/Makefile.am	2008-03-29 10:20:42.000000000 +0200
+++ kazehakase-0.5.4/src/Makefile.am	2008-05-16 00:51:59.000000000 +0300
@@ -203,6 +203,7 @@
 	$(GTK_LIBS) \
 	$(SM_LIBS) \
 	$(LIBGNUTLS_LIBS) \
+	$(LIBGCRYPT_LIBS) \
 	-lz		\
 	$(libraries)
 
